roles/matrix-bridge-mautrix-signal/defaults/main.yml

```diff
@@ -127,3 +127,5 @@ matrix_mautrix_signal_registration_yaml: "{{ lookup('template', 'templates/regis
 matrix_mautrix_signal_registration: "{{ matrix_mautrix_signal_registration_yaml|from_yaml }}"
 
 matrix_mautrix_signal_log_level: 'DEBUG'
+
+matrix_mautrix_signal_enable_encryption: false
```

Comment:
This should be called `matrix_mautrix_signal_bridge_encryption_allow` to match the configuration key.

Reply:
:+1: fixed